product/views.py

```diff
@@ -0,0 +1,43 @@
+import json
+from random                     import uniform
+
+from django.views               import View
+from django.http                import JsonResponse
+from django.core.exceptions     import ValidationError
+
+from product.models             import Product
+
+class ProductDetailView(View):
+    def get(self ,request):
```

Comment:
```suggestion
    def get(self ,request, product_id):
```
path parameter로 상품의 id를 받기 위해서는 다음과 같이 수정이 필요합니다.